### PR TITLE
Add tests for deprecated commands in RHEL-9

### PR DESCRIPTION
**Pull request by velezd**
These new tests check that correct warning message is shown when deprecated command/option is used and rest of the command still works.

There is a potential issue that should be cleared up before merging: https://bugzilla.redhat.com/show_bug.cgi?id=1959486

### CODE REVIEW

**Review by VladimirSlavik — COMMENTED** [review]
As Radek says, the question of granularity is something to further discuss. From the developer perspective, this should be a single test "deprecated things in RHEL 9.x". No need to waste cycles on checking one line each in a different run. From the QE perspective, I can see how parity is more important: one change = one test.

Defining how/where, when, which tests run might be a prerequisite for that discussion, though?

Otherwise LGTM, of course :)

**Comment by velezd** [comment]
I didn't find hard reason to keep them separate and it makes sense to save resources where possible.
So I combined them to two tests, timezone --nontp and --ntpservers should stay separate.

File: deprecated-pwpolicy.ks.in [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+%end
+
+%post --nochroot
+if ! $(grep -q "KickstartDeprecationWarning: The %%anaconda section has been deprecated" /tmp/anaconda.log); then
+    echo 'Deprecation warning is missing for %anaconda' > /mnt/sysroot/root/RESULT
+fi
+if ! $(grep -q "KickstartDeprecationWarning: The pwpolicy command has been deprecated" /tmp/anaconda.log); then
+    echo 'Deprecation warning is missing for pwpolicy' > /mnt/sysroot/root/RESULT
+fi
+%end
```
[the remark this answers is not in the document]
Comment (velezd) [reply deprecated-pwpolicy.ks.in]: From my point of view that is complete different issue, anaconda adds it to the kickstart regardless of user using the command or not.

**Review by jkonecny12 — APPROVED** [review]
Looks good to me.

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

**Review by rvykydal — APPROVED** [review]
Looks good to me, thank you.

**Pull request merged**